test/gtest/multi_threading.cpp

```diff
@@ -28,10 +28,12 @@ class MultiThreadingTestFixture : public testing::Test {
     uintptr_t addr = 0;
     size_t len = 1024;
     uint64_t dev_id = 0;
+    std::string local_agent_name = "test_agent";
+    std::string remote_agent_name = "remote_agent";
```

Comment:
```C++
    constexpr static std::string_view local_agent_name = "test_agent";
    constexpr static std::string_view remote_agent_name = "remote_agent";

    nixlAgent createAgent(std::string_view name) {
```

Reply:
I much rather prefer simple and obvious _test_ code.

Reply:
The specifiers make the code more obvious.
`constexpr` makes it obvious that the variable is a constant and can be evaluated at compile time.

`std::string_view` almost the same thing as `const std::string&`, but might be faster in some cases. So why not use.

Reply:
Because it is a test code that I would prefer to keep it simple and readable. No one cares about this test class being initialized in 1microsecond faster.

Reply:
I see... Thanks for checking this.